star_tides/core/actions/login_user_action.py

```diff
@@ -81,7 +92,10 @@ def run(self):
                 user = UserModel.objects(email=email).first()
                 if user is None:
                     # TODO create a custom exception and raise it
-                    raise Exception('User failed to create')
+                    raise GenericActionError(
```

Comment:
Up to you: make a DatabaseError? Regardless, this would probably be a 500. If I understand this code correctly, you're looking for a user with the given email. That would probably make more sense as a NotFoundError. Update the error message too?